pootle_pytest/fixtures/site.py

```diff
@@ -20,9 +20,16 @@ def site_matrix(request, system, settings):
         TranslationProjectFactory, StoreFactory, UnitFactory)
 
     from pootle_store.models import UNTRANSLATED, TRANSLATED, FUZZY, OBSOLETE
+    from pootle_app.models import Directory
 
-    # create root and projects directories
-    DirectoryFactory(name="projects", parent=DirectoryFactory(parent=None))
+    # create root and projects directories, first clear the class cache
```

Comment:
I was told several times that I was rude. Perhaps it is a cultural thing, but I see @julen comments to be direct and clear, but not necessarily rude. As I said, perhaps it is cultural.